example/viam_example_app/lib/main.dart

```diff
@@ -119,11 +124,22 @@ class _MyHomePageState extends State<MyHomePage> {
     if (!_isNavigable(rname)) {
       return null;
     }
+    if (rname.subtype == Motor.subtype.resourceSubtype) {
+      return MotorScreen(motor: Motor.fromRobot(_robot, rname.name), resourceName: rname);
+    }
     if (rname.subtype == Camera.subtype.resourceSubtype) {
-      return StreamScreen(camera: Camera.fromRobot(_robot, rname.name), client: _getStream(rname), resourceName: rname);
+      return StreamScreen(
+          camera: Camera.fromRobot(_robot, rname.name),
+          client: _getStream(rname),
+          resourceName: rname);
+    }
+    if (rname.subtype == Motor.subtype.resourceSubtype) {
```

Comment:
This looks duplicative with the code a couple lines up. Am I missing some dart-ism?

Reply:
Good catch, thank you! This is duplicate code that needs to be removed. will push up change momentarily... 🙏

Reply:
I decided to remove the higher, preferring alphabetization.